client/src/plugins/element-templates-modal/modeler/EditorActions.js

```diff
@@ -9,50 +9,71 @@
  */
 
 export default class EditorActions {
-  constructor(commandStack, editorActions, selection) {
-    editorActions.register('applyElementTemplate', elementTemplate => {
-      const selectedElements = selection.get();
+  constructor(commandStack, editorActions, selection, canvas) {
 
-      if (selectedElements.length !== 1) {
-        return false;
-      }
+    // Register action to apply an element template to the selected element
+    editorActions.register('applyElementTemplate', elementTemplate => {
+      const element = getSelectedElement();
 
-      const element = selectedElements[ 0 ];
+      if (element) {
+        commandStack.execute('propertiesPanel.camunda.changeTemplate', {
+          element,
+          newTemplate: elementTemplate
+        });
 
-      commandStack.execute('propertiesPanel.camunda.changeTemplate', {
-        element,
-        newTemplate: elementTemplate
-      });
+        return true;
+      }
 
-      return true;
+      return false;
     });
 
+    // Register action to get the currently selected element
     editorActions.register('getSelectedElement', () => {
-      const selectedElements = selection.get();
+      return getSelectedElement();
+    });
 
-      if (selectedElements.length !== 1) {
-        return null;
+    // Register action to get the template applied to the currently selected element
+    editorActions.register('getSelectedElementAppliedElementTemplate', () => {

```

Comment:
I find the naming a bit heavy, but then again I'm not sure what could fit...
Maybe 'getTemplateAppliedToSelectedElement' ?

Reply:
Ah it existed prior to this PR